sql/pre_install/tables.sql

```diff
@@ -195,8 +195,14 @@ CREATE TABLE IF NOT EXISTS _timescaledb_config.bgw_job (
     job_type            NAME        NOT NULL,
     schedule_interval   INTERVAL    NOT NULL,
     max_runtime         INTERVAL    NOT NULL,
-    max_retries         INT         NOT NULL,
+    max_retries         INTEGER     NOT NULL,
     retry_period        INTERVAL    NOT NULL,
+    proc_name           NAME        NOT NULL DEFAULT '',
```

Comment:
why is there a default empty string? Why not simply?
```suggestion
    proc_name           NAME       ,
```

Reply:
NULL columns dont go well with some of the catalog C functions

Reply:
made hypertable_id and config NULL, all the other columns with DEFAULT will get the default removed once all policies are migrated